TimeSeriesPoint a
TimeSeriesPoint b
---
TimeSeriesPoint sum
